pkg/bootstrap/platform/aws.go

```diff
@@ -98,7 +98,7 @@ func getAWSInfo(path string, ipv6 bool) (string, error) {
 
 	resp, err := http.DoHTTPGetWithTimeout(url, time.Millisecond*100)
 	if err != nil {
-		log.Errorf("error in getting aws info for %s : %v", path, err)
+		log.Warnf("error in getting aws info for %s : %v", path, err)
```

Comment:
prefer Debugf

Reply:
Sure. I didn't know how to decide on warn or debug. 
I see that azure and gcp have log levels set to warn though..

Reply:
? If set to warn, it will print a confusing log in other platform

Reply:
Sorry, warn messages are logged only if the cloud platform is explicitly specified.
During the discovery phase (when cloud platform not set), it is good to have these logs at debug level. Agree.